themes/mirror/static/js/article.js

```diff
@@ -185,7 +185,7 @@ $(document)
     var latestBottomTemplate = $.templates("#latest-bottom-tempalte");
 
 
-    $.getJSON( "https://crossorigin.me/https://storage.googleapis.com/dev-site/json/latest-posts.json", function( data ) {
+    $.getJSON( "http://stage.mirrormedia.mg/news/json/latest-posts.json", function( data ) {
```

Comment:
However, we can not use variables in script file, even it's not inside the static folder.